pandas/tests/base/conftest.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+import pytest
+
+import pandas as pd
+from pandas.tests.indexes.conftest import indices_dict
+
+
+def _create_series(index):
+    """ Helper for the _series dict """
+    data = np.random.randn(len(index))
+    return pd.Series(data, index=index, name=index.name)
+
+
+_series = {
+    f"series-with-{i_id}-index": _create_series(i) for i_id, i in indices_dict.items()
+}
+
+
+def _create_narrow_series(data_dtype):
```

Comment:
just call this dtype, use the is_integer and so one for comparisons; we really try to avoid using numpy things generally (as they don't scale to all of our dtypes).

Reply:
@jreback I just tried that, but `is_integer(np.int16)` is actually `False`. Similar for the other dtypes or `is_float` 😐